frontend/device/sdl/device.lua

```diff
@@ -80,16 +62,12 @@ local Device = Generic:new{
     openLink = function(self, link)
         local enabled, tool = getLinkOpener()
         if not enabled or not tool or not link or type(link) ~= "string" then return end
-        if jit.os == "OSX" then
-            return os.execute(tool .. " '" .. link .. "'") == 0
-        else
-            return os.execute('env -u LD_LIBRARY_PATH '..tool.." '"..link.."'") == 0
-        end
+        return runCommand(tool .. " '" .. link .. "'")
     end,
     canExternalDictLookup = yes,
     getExternalDictLookupList = getExternalDicts,
     doExternalDictLookup = function(self, text, method, callback)
-        external_dict_when_back_callback = callback
+        external_dicts_callback = callback
```

Comment:
I don't remember much how that works - but if you're just renaming it for aesthetics, and if it is still really called only when back from external dict, add this fact as a comment so a future reader knows that. (Unless it's mentionned elsewhere).

Reply:
Thank you! I will leave it out the equation for now. I have a branch with refactored 3rd party app integration which I plan to PR soon.